### PR TITLE
feat(form.vue): adds additional message support

### PR DESCRIPTION
**Pull request by IainSAP**
- adds additional messages under prompts with severity icon and color
- adds tests to cover
- updated sample-app

image
image

### CODE REVIEW

File: __tests__/list.spec.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('Question of type list', () => {
     new Vue({ vuetify });
 
     document.body.setAttribute('data-app', 'true');
-    const wrapper = mount(Form, { vuetify, attachToDocument: true });
+    const wrapper = mount(Form, { vuetify, attachTo: document.body });
```
Comment (IainSAP) [inline __tests__/list.spec.js:140]: Updates deprecated usage

File: __tests__/list.spec.js [hunk 2]
```diff
@@ -175,21 +175,21 @@ describe('Question of type list', () => {
...
+    const menu = wrapper.findComponent({name: 'v-menu'});
     menu.trigger('click');
     await Vue.nextTick();
 
-    const list = wrapper.find({name: 'v-select-list'});
-    expect(list.findAll({name:'v-list-item'})).toHaveLength(6);
-    expect(list.findAll({name:'v-divider'})).toHaveLength(1);
-    expect(list.findAll({name:'v-subheader'})).toHaveLength(1);
+    const list = wrapper.findComponent({name: 'v-select-list'});
+    expect(list.findAllComponents({name:'v-list-item'})).toHaveLength(6);
```
Comment (IainSAP) [inline __tests__/list.spec.js:188]: Updates deprecated usage, see: https://v1.test-utils.vuejs.org/api/wrapper/#find

**Review by alex-gilin — COMMENTED** [review]
LGTM. 
Generally, its an independent package. I've some doubts about the coupling with the `@sap-devx/yeoman-ui-types` package. @idoprz, do you know other usages for this package besides `yeoman-ui`?

**Comment by idoprz** [comment]
@IainSAP / @alex-gilin  we do use the inquirer.js gui in other non yeoman related projects. You should not add dependencies from inquirer.js gui to yeoman ui.

**Comment by IainSAP** [comment]
> @IainSAP / @alex-gilin we do use the inquirer.js gui in other non yeoman related projects. You should not add dependencies from inquirer.js gui to yeoman ui.

@idoprz Can you please suggest how we can share the "Severity" type otherwise. Should there be a new module created containing types required by inquirer-gui that can be imported by YUI consumers also as there is a need for shared types between these modules. I think there will be more in future.

**Review by alex-gilin — APPROVED** [review]
LGTM

**Pull request merged**